app.py

```diff
@@ -34,26 +33,18 @@
 _LOGGER = logging.getLogger(__title__)
 
 
-@click.command()
-@click.option(
-    "--repositories", help="Repositories to be analysed (e.g thoth-station/performance)",
-)
-@click.option(
-    "--organizations",
-    "-o",
-    type=str,
-    required=False,
-    help="Organizations (all of their repositories) to be analysed (e.g. AICoE)",
-)
-def main(
-    repositories: Optional[str], organizations: Optional[str],
-):
-    """Command Line Interface for SrcOpsMetrics."""
-    gh = Github()
+def main():
+    """MI-Scheduler entrypoint."""
+    oc = OpenShift()
+    cm = oc.get_configmap(configmap_id="mi-scheduler", namespace="thoth-test-core")
 
-    orgs = organizations.split(",") if organizations is not None else []
+    organizations = cm["ORGANIZATIONS"]
+    repositories = cm["REPOSITORIES"]
```

Comment:
arent these value lowercased? https://github.com/thoth-station/thoth-application/blob/master/mi-scheduler/base/configmaps.yaml

also lets use get() method, so we dont run into dict errors. WDYT?
```suggestion
    organizations = cm.get("organizations", "")
    repositories = cm.get("repositories", "")
```

Reply:
nice catch!